.eslintrc.js

```diff
@@ -32,4 +32,58 @@ module.exports = {
     '@next/next/no-img-element': ['off'],
     'linebreak-style': ['error', os.platform() === 'win32' ? 'windows' : 'unix'],
   },
+  overrides: [
+    {
+      files: ['**/*.ts', '**/*.tsx'],
+      env: {
+        es2021: true,
+        node: true,
+        jest: true,
+      },
+      extends: [
+        'next',
+        'airbnb',
+        'airbnb-typescript',
+        'plugin:cypress/recommended',
+        'eslint:recommended',
```

Comment:
do we need to add `eslint:recommended` what benefits are we getting that we weren't getting already?